coollabsio/coolify
# Enhancement: Preserve scroll position in navbar to improve UX

## Summary
This PR enhances the user experience by preserving the navbar scroll position during Livewire updates. This prevents the navbar from resetting its scroll position, which can be disorienting for users (I am them).

## Changes
- Added Alpine.js functionality to save and restore the navbar scroll position before and after Livewire updates (ln 48).

## Testing Instructions
1. Navigate through the application using the navbar links.
2. Ensure that the scroll position of the navbar is maintained even after Livewire updates.

## Notes
I noticed this improvement while using Coolify for the first time and thought it would enhance the user experience. Since I am new to using this platform, contributing, using the tool and working on the stack, I would appreciate it if someone could verify these changes and confirm that they work as expected. That can be easily done by opening the main page (don't know how you call it, sorry), scrolling the navbar down and clicking into an item, if the navbar resets to the beginning then I'm sad, but thank you, kind soul, in advance.
